.github/workflows/build.yml

```diff
@@ -4,11 +4,10 @@ on: [push, pull_request]
 
 env:
   COUCH_URL: http://admin:pass@localhost:5984/medic-test
-  BUILDS_SERVER: ${{ secrets.AUTH_MARKET_URL && '_couch/builds_testing' || '_couch/builds_external' }}
-  STAGING_SERVER: ${{ secrets.AUTH_MARKET_URL && '_couch/builds_4' || '_couch/builds_external' }}
+  BUILDS_SERVER: ${{ '_couch/builds_external' }}
+  STAGING_SERVER: ${{ '_couch/builds_external' }}
```

Comment:
This would mean that we never publish builds as we did before, right?

Reply:
Please ignore that... the first commit was to turn myself into an external contributor so I could reproduce the issue - it'll be reverted before review!